fix: update readme for free zeit accounts (#109)
zeit/og-image: README.md

4. Make changes by swapping out images, changing colors, etc (see [contributing](https://github.com/zeit/og-image/blob/master/CONTRIBUTING.md) for more info)
5. Run locally with `now dev` and visit [localhost:3000](http://localhost:3000)  (if nothing happens, run `npm install -g now`)
6. Deploy to the cloud by running `now` and you'll get a unique URL
7. Setup [GitHub](https://zeit.co/github) to autodeploy on push

Alternatively, you can do a one-click to deploy with the button below.

[![Deploy to now](https://zeit.co/button)](https://zeit.co/new/project?template=zeit/og-image)

> If you are using free zeit account you may need to remove all configuration inside `now.json` besides

Reviewer: ```suggestion
If you are using free ZEIT Now plan, you will need to remove all configuration inside `now.json` besides
```
